### ISSUE TITLE
Reduced speech-in-noise reception

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11683]
# Reduced speech-in-noise reception

## Proposed definition

An abnormal speech discrimination characterized by reduced ability to understand speech in the presence of background noise.

## Parent term

- `HP:0001963 Abnormal speech discrimination`

## Synonyms

- Impaired speech-in-noise perception

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617
@evanc1648

**Comment by pnrobinson** [comment]
@evanc1648

Adding new term: 
- ID: HP:6001525
- label: Impaired speech-in-noise perception
- parents: [Abnormal speech discrimination (HP:0001963)](https://hpo.jax.org/app/browse/term/HP:0001963)
- pmid: PMID:39788732|PMID:42517808
- def: Reduced ability to understand speech in the presence of background noise.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11683
- synonyms: Reduced speech-in-noise reception(NONE)
